frontend/src/components/evaluation/allocs.js

```diff
@@ -1,6 +1,6 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
-import AllocationList from '../allocations'
+import AllocationList from '../allocation_list'
```

Comment:
best practice to have component and filename match, makes it much more predictable and easy to find what you are looking for as the project grow :)